applications/CompressiblePotentialFlowApplication/compressible_potential_flow_application.h

```diff
@@ -101,8 +101,8 @@ class KRATOS_API(COMPRESSIBLE_POTENTIAL_FLOW_APPLICATION) KratosCompressiblePote
 
     const IncompressiblePotentialFlowElement<2,3> mIncompressiblePotentialFlowElement2D3N;
     const CompressiblePotentialFlowElement<2,3> mCompressiblePotentialFlowElement2D3N;
-    const AdjointPotentialFlowElement<IncompressiblePotentialFlowElement<2,3>> mAdjointIncompressiblePotentialFlowElement2D3N;
-    const AdjointPotentialFlowElement<CompressiblePotentialFlowElement<2,3>> mAdjointCompressiblePotentialFlowElement2D3N;
+    const AdjointFiniteDifferencePotentialFlowElement<IncompressiblePotentialFlowElement<2,3>> mAdjointIncompressiblePotentialFlowElement2D3N;
```

Comment:
When defining the element in the application now we use the FiniteDifference element